tools/envoy_headersplit/headersplit_test.py

```diff
@@ -0,0 +1,103 @@
+# Lint as: python3
```

Comment:
I'm having trouble getting this test to run locally or in CI.  See https://dev.azure.com/cncf/4684fb3d-0389-4e0b-8251-221942316e06/_apis/build/builds/63187/logs/48 from PR https://github.com/envoyproxy/envoy/pull/14615 which tries to add //tools/... to CI

Do you know what may be going wrong?

======================================================================
ERROR: test_class_definitions (__main__.HeadersplitTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/b/f/w/bazel-out/k8-opt/bin/tools/envoy_headersplit/headersplit_test.runfiles/headersplit_pip3_pypi__clang_10_0_1/clang/cindex.py", line 4172, in get_cindex_library
    library = cdll.LoadLibrary(self.get_filename())
  File "/usr/lib/python3.6/ctypes/__init__.py", line 426, in LoadLibrary
    return self._dlltype(name)
  File "/usr/lib/python3.6/ctypes/__init__.py", line 348, in __init__
    self._handle = _dlopen(self._name, mode)
OSError: libclang-10.so: cannot open shared object file: No such file or directory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/b/f/w/bazel-out/k8-opt/bin/tools/envoy_headersplit/headersplit_test.runfiles/envoy/tools/envoy_headersplit/headersplit_test.py", line 55, in test_class_definitions
    idx = Index.create()
  File "/b/f/w/bazel-out/k8-opt/bin/tools/envoy_headersplit/headersplit_test.runfiles/headersplit_pip3_pypi__clang_10_0_1/clang/cindex.py", line 2698, in create
    return Index(conf.lib.clang_createIndex(excludeDecls, 0))
  File "/b/f/w/bazel-out/k8-opt/bin/tools/envoy_headersplit/headersplit_test.runfiles/headersplit_pip3_pypi__clang_10_0_1/clang/cindex.py", line 212, in __get__
    value = self.wrapped(instance)
  File "/b/f/w/bazel-out/k8-opt/bin/tools/envoy_headersplit/headersplit_test.runfiles/headersplit_pip3_pypi__clang_10_0_1/clang/cindex.py", line 4146, in lib
    lib = self.get_cindex_library()
  File "/b/f/w/bazel-out/k8-opt/bin/tools/envoy_headersplit/headersplit_test.runfiles/headersplit_pip3_pypi__clang_10_0_1/clang/cindex.py", line 4177, in get_cindex_library
    raise LibclangError(msg)
clang.cindex.LibclangError: libclang-10.so: cannot open shared object file: No such file or directory. To provide a path to libclang use Config.set_library_path() or Config.set_library_file().